lib/file.js

```diff
@@ -102,11 +102,12 @@ File.prototype.dependencies = function () {
  */
 
 File.prototype.load = function *() {
-  if (this.src) return this;
+  if (typeof this.src !== 'undefined') return this;
```

Comment:
this is just my ocd, but why use `typeof` rather than just comparing with `undefined`?  the latter is terse and [slightly faster](https://gist.github.com/stephenmathieson/ad70f8d680dcece6f40f)